sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/DateTimeUtils.scala

```diff
@@ -906,4 +906,45 @@ object DateTimeUtils {
     val days = period.getDays
     new CalendarInterval(months, days, 0)
   }
+
+  /**
+   * Since the Proleptic Gregorian calendar is de-facto calendar worldwide, as well as the chosen
```

Comment:
In Spark 3.0, we switch to the Proleptic Gregorian calendar and use `DateTimeFormatter` for parsing/formatting datetime values. The pattern string is incompatible with the one defined by `SimpleDateFormat` in Spark 2.4 and earlier. This function ...

Reply:
Thanks, done in 5382508